### PR TITLE
Added bounded counter CRDT.

### PR DESCRIPTION
**Pull request by lythq**
Also fixed error handling in transactions and downstream generation, as well as rename to Antidote in riak_test setup instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,11 +230,11 @@ Running Tests
 2. cd RIAK_TEST and run commands
 	* make
 5. Download and put `.riak_test.config` in home directory from following link
-	https://gist.github.com/cmeiklejohn/d96d7167e1a8ed4b128d
+	https://gist.github.com/lythq/f979e11749a868fc33dd
```
Comment (cmeiklejohn) [inline README.md]: Care to clarify the changes between the two gists?

Comment (lythq) [inline README.md]: Yours still has the project name floppystore, but riak_test on Antidote is named antidote.
So you get an error if you haven't run riak_test on the previous code, and the new code isn't the one being tested.
I also added the test_paths setting, so you're less prone to skip that step.

Comment (cmeiklejohn) [inline README.md]: Great.  But, why even bother having it in a gist anymore?  How about just put the file in this repo?

Comment (lythq) [inline README.md]: Well pointed out. Automated that part and removed the gist.

**Comment by bieniusa** [comment]
@cmeiklejohn I want to merge this pull request. Just one doubt: What about the Basho copyright notice? We are slowly moving from state-based to op-based. Would the copyright notice still apply?

**Comment by lythq** [comment]
This CRDT isn't based on the ones from Riak, it was built from scratch only following the guidelines from other modules.

**Comment by bieniusa** [comment]
Ok, let’s remove the copyright notice then.

**Comment by bieniusa** [comment]
Just checked again. The pn_counter contains a copyright notice for basho - this one should go I think.

**Comment by lythq** [comment]
pn_counter has nothing to do with this CRDT or this pull request, except for both being CRDTs and following the same guidelines...

**Pull request merged**